index.html

```diff
@@ -20,9 +20,9 @@
     <link rel="icon" type="image/png" sizes="96x96" href="./img/favicon-96x96.png">
     <link rel="icon" type="image/png" sizes="16x16" href="./img/favicon-16x16.png">
     <link rel="manifest" href="./img/manifest.json">
-    <meta name="msapplication-TileColor" content="#ffffff">
+    <meta name="msapplication-TileColor" content="#000000">
```

Comment:
I googled the code and it seems that it is black %) could you please change it back to white as it is the home page?

Reply:
Done, thx